entity-service/src/main/kotlin/com/egm/stellio/entity/EntityServiceApplication.kt

```diff
@@ -8,6 +8,7 @@ import org.springframework.scheduling.annotation.EnableAsync
 @EnableAsync
 class EntityServiceApplication
 
+@Suppress("SpreadOperator")
```

Comment:
And this tag

Reply:
it disables the spread operator warning for this method (https://detekt.github.io/detekt/performance.html#spreadoperator), which is not really relevant here.